packages/join-flow/yarn.lock

```diff
@@ -5674,6 +5674,11 @@ prepend-http@^2.0.0:
   resolved "https://registry.npmjs.org/prepend-http/-/prepend-http-2.0.0.tgz"
   integrity sha1-6SQ0v6XqjBn0HN/UAddBo8gZ2Jc=
 
+prettier@^3.3.3:
```

Comment:
Do you want to add this to some kind of pre-commit?

Reply:
No it's just for my IDE to find it.